src/templates/community-champion-profile.js

```diff
@@ -27,16 +82,194 @@ const StyledBreadcrumb = styled(Breadcrumb)`
     > a {
         font-size: ${fontSize.xsmall};
     }
-    @media ${screenSize.upToLarge} {
+    @media ${screenSize.upToSmallDesktop} {
         margin-bottom: ${size.large};
     }
 `;
 
+const Quote = styled(P)`
+    background: ${({ theme }) => theme.gradientMap.magentaSalmonSherbet};
+    background-clip: text;
+    font-family: 'Fira Mono';
+    font-size: 20px;
+    font-weight: 500;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
```

Comment:
took some of the properties from the gradient text component [here!](https://github.com/mongodb/devhub/blob/staging/src/components/pages/academia/students-educators-details.js#L94)